lib/asn1numericstring.dart

```diff
@@ -0,0 +1,40 @@
+part of asn1lib;
+
+///
+/// An ASN1 Numeric String
+///
+/// A NumericString is a restricted character string type in the ASN.1 notation.
+/// It is used to describe data that does only consist of digits and space character.
+///
+class ASN1NumericString extends ASN1Object {
+  // The decoded string value
+  late String stringValue;
+
+  ///
+  /// Create an [ASN1NumericString] initialized with String value.
+  ///
+  /// Optionally override the tag
+  ///
+  ASN1NumericString(this.stringValue, {int tag = NUMERIC_STRING_TYPE})
+      : super(tag: tag);
+
+  ///
+  /// Create an [ASN1PrintableString] from an encoded list of bytes.
```

Comment:
PrintableString or NumericString? Just checking

Reply:
It was just me being inattentive to the copy-pasted comment. Good catch!